server.js

```diff
@@ -0,0 +1,51 @@
+const file404 = require('./404.js');
+const helium = require('./helium.js');
+const hydrogen = require('./hydrogen.js');
+const index = require('./index.js');
+const styles = require('./styles');
+const net = require('net');
+
+let contentType = "text/html; charset=utf-8";
+let body = "";
+let date = "";
+
+// this creates a server
+const server = net.createServer((socket) => {
+  socket.setEncoding('utf8');
+  socket.on('data', (data) => {
+    let URI = data.slice(4, data.indexOf("H") - 1);
+
+    if (URI === "/" || URI === "/index.html") {
+      body = index.index;
```

Comment:
For all these files, (index, hydrogen, helium, etc) you could:

`module.exports = hydrogen;`

Then you don't need to do `hydrogen.hydrogen` in this file.